bbb-common-web/src/main/java/org/bigbluebutton/presentation/imp/ExternalProcessExecutor.java

```diff
@@ -79,12 +77,35 @@ public boolean exec(List<String> cmd, Duration timeout) {
 			return false;
 		}
 
+		// Capture error output
+		StringBuilder errorOutput = new StringBuilder();

```

Comment:
I am concerned that collecting the log output of the command in memory in the bbb-web process is a bad idea. A malicious input might cause the pdf conversion to generate a very large amount of log output, causing a denial of service due to OOM in this code.

In similar-looking code in the recording system (where I deal with ffmpeg logs), I set up the output of the spawned command to be written to a temporary file, and then read from the file after the process exits to incorporate the command output into the logs.

Reply:
Now it's using the existing `AbstractCommandHandler extends NuAbstractProcessHandler`.
If it is still an issue even using NuProcess, I think we should address in a different PR (as it is also used by other components).